src/components/creation/RaceCreation/RaceCard.tsx

```diff
@@ -4,65 +4,96 @@ import RacesData from "../../../api/RacesData";
 import Race from "../../../model/Character/Races/race.interface";
 import RaceModal from "./RaceModal";
 import '../../styles/raceCreate.css'
-import { PropsFor } from "@mui/system";
 import SelectedRacePage from "./SelectedRacePage";
-import { Opacity } from "@mui/icons-material";
 
 const racesData = new RacesData()
 
+class RaceDesc {
+    title!: string;
+    desc: string[] = []
+}
+class AstrickTrait {
+    title!: string;
+    desc: string[] = []
+}
+class MultiAstrickTrait {
+    astrickTraits: AstrickTrait[] = []
+}
+
 function RaceCard(props: any) {
-    const [prop, setProp] = useState<string | undefined>()
+    const [raceSlug, setRaceSlug] = useState<string | undefined>()
     const [clicked, setClicked] = useState(false)
     const { data, status } = useQuery(['races'], racesData.fetchRaces)
+    const [raceData, setRaceData] = useState<AstrickTrait[]>([])
+    const [raceDesc, setRaceDesc] = useState<RaceDesc>()
 
     const handleClick = (race: Race) => {
-        setProp(race.slug)
+        setRaceSlug(race.slug)
+        props.setRaceSlug(race.slug)
         setClicked(true)
     }
 
     
     const handleClose = () => {setClicked(false)};
+   console.log(raceSlug)
     return (
+        <>
+        {props.race === "" ? (
        <div className="race-title-content-container">
-                <div>
-                        <h1
-                            style={{
-                                opacity: '.6'
-                            }}>
-                            Choose a Race
-                        </h1>
-                </div>
-                <div style={{ border: '1px black solid', width: '50%', margin: '5px'}}></div>
-                <div
-                    style={{
-                        margin: '5px',
-                        width: '75%'
-                    }}>
-                        {data?.results.map((res: Race) => (
-                            <IndvRace res={res} handleClick={handleClick}/>
-                        ))}
-                </div>
+       <div>
+               <h1
+                   style={{
+                       opacity: '.6'
+                   }}>
+                   Choose a Race
+               </h1>
        </div>
-    )
-}
+       <div style={{ border: '1px black solid', width: '50%', margin: '5px'}}></div>
```

Comment:
I typically don't leave empty tags and instead will close them like so <div style=***** />